### PR TITLE
Adds commands to check User Profile Disk path

### PR DESCRIPTION
**Pull request by allez-allez-allez**
The check is performed after `join-domain` command.

### CODE REVIEW

File: modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml [hunk 1]
```diff
@@ -619,6 +620,23 @@ Resources:
...
                 - PowershellCommand: !FindInMap [ShellCommandMap, powershell, command]
               waitAfterCompletion: forever
+        check-upd-path:
+          commands:
+            10-check-path:
+              command: !Sub
+                - >-
+                  ${PowershellCommand} -Command "
+                  Invoke-Command -ScriptBlock {
+                  Write-Host 'Attempting to verify UPD path ${UserProfileDiskPath}';
```
Comment (lorengordon) [inline modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml]:
```suggestion
                  Write-Verbose 'Attempting to verify UPD path ${UserProfileDiskPath}';
```

File: modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml [hunk 2]
```diff
@@ -619,6 +620,23 @@ Resources:
...
+                  if (-Not $(Test-Path ('filesystem::' + '${UserProfileDiskPath}'))){
+                  throw 'User Profile Disk Path does not exist. Abort!'
+                  } Write-Host 'UPD path ${UserProfileDiskPath} exists. Continue...'
```
Comment (lorengordon) [inline modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml]:
```suggestion
                  } Write-Verbose 'UPD path ${UserProfileDiskPath} exists. Continue...'
```

**Comment by allez-allez-allez** [comment]
@lorengordon, updated the template based on your feedback. Using `Invoke-RetryCommand` to invoke the `Test-Path` command. Tested and it seems to be working.

**Comment by moskey71** [comment]
> @lorengordon, updated the template based on your feedback. Using `Invoke-RetryCommand` to invoke the `Test-Path` command. Tested and it seems to be working.

Did you test with a valid and invalid path?

**Comment by allez-allez-allez** [comment]
@moskey71 invalid path test was a success. I'm trying to test with a valid path..it's been giving me problems. I will post the result back when complete.

**Comment by allez-allez-allez** [comment]
@moskey71 ok valid path test passed. Merging now

**Pull request merged**